model_compression_toolkit/common/post_training_quantization.py

```diff
@@ -137,6 +137,7 @@ def post_training_quantization(in_model: Any,
                         bit_widths_config)
 
     common.Logger.info(f'Approximated model size (in bytes): {tg.get_memory()}')
+    common.Logger.info(f'Approximated compression ratio: {round(tg.get_float_memory() / (tg.get_memory() + 1e-8), 3)}')
 
```

Comment:
I think we need to take the float memory of the original graph (retrieved from the model reader) instead of tg.

Reply:
Why do you think so?